index.js

```diff
@@ -66,6 +66,8 @@ class ResembleHelper extends Helper {
       resemble.outputSettings({
         boundingBox: options.boundingBox,
         ignoredBox: options.ignoredBox,
+        boundingBoxes: options.boundingBoxes,
```

Comment:
@mlincoln4 this change is probably going to impact https://github.com/codeceptjs/codeceptjs-resemblehelper/blob/master/index.js#L379 could you please make sure it works as expected

Reply:
see where we set the boundingBox option, https://github.com/codeceptjs/codeceptjs-resemblehelper/blob/1cc3b604e6f99d91e6c4900c6046bc50f9bc304f/index.js#L322

Reply:
Well that is checking the visual difference for an element, in which case I don't think bounding boxes need to be supported since you are already specifying an element for the bounds. I am not sure though. That part isn't as important to me, I just happened to notice the plural ones weren't currently supported.